x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -56,6 +56,20 @@ export const alertMappings: SavedObjectsTypeMappingDefinition = {
           enabled: false,
           type: 'object',
         },
+        frequency: {
+          type: 'nested',
```

Comment:
Q: Do we need the `type: 'nested'`? Looking at the code, it looks like it's an object (not an array of objects).

Also, it doesn't look like we are querying these fields (apologies if we are, and I couldn't find it). Do you think it's worth adding [`index: false`](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-index.html) to avoid unnecessarily increasing the number of fields in the mappings?

Also, do we need a migrations script to add this field to previous documents that don't need it?

Reply:
I forgot removing that "nested" type! Thank you for finding it out! 
Fixed with: https://github.com/elastic/kibana/pull/146183/commits/707875ff1a47274d6b32623b83568d2f6d33afa6

We use it in the execution_handler. https://github.com/elastic/kibana/pull/146183/files#diff-8656f6dc8acf27d0e712d9c277de92a693b6e2ee3bcdae9f08f8395141f7b38c

I am not sure about adding, `index: false` let us discuss it. Then i may add that option.

No migration is needed, the code fallbacks to the old field if this one does not exist :)